### ISSUE TITLE
command 'extension.NyanCatReload' not found on windows

### ISSUE THREAD

**Issue by dmikucos** [zhengrenzhe/vscode-Nyan-Cat#4]
So I installed it. And When I use shift + command + p and search for Nyan Cat: refresh status bar, it does not work.

ples help I need my cats :(

**Comment by Nalleyer** [comment]
+1, maybe I have the same issue, what I see is:
`command 'extension.NyanCatRefresh' not found`

**Comment by dmikucos** [comment]
Exactly that, the extention needs to be updated if i'm not mistaking

**Comment by FloWzoW** [comment]
im having the same problem, it doesnt work :(

**Comment by volotovskyy** [comment]
I can confirm it to on mac OS :(

**Comment by cig0** [comment]
Hi all, same thing on GNU+Linux

**Comment by SocketNet** [comment]
+1 Windows 10 vscode version: 1.41.1

**Comment by hikari6688** [comment]
with the same problem,need my cat!

**Comment by cesargroening** [comment]
Same problem, i had nyan cat on emacs, i want it in visual code, help! :C

**Comment by cesargroening** [comment]
Recently i saw the source code and i found that almost every line of code was commented, and the line that set the image in the bar is not working anymore,  i think it worked for old versions of visual code but not in news one

Do anyone know a way to put images on the bar, that still working in new versions, i'm thinking in do a new nyan-cat complement?

**Comment by yuzhen1024** [comment]
me too

**Comment by zhengrenzhe** [comment]
@cig0 @FloWzoW @SocketNet @Nalleyer @cesargroening @volotovskyy @yuzhen1024 @cesargroening @hikari6688 @ToadallyAwesome2 @Nalleyer 

new version 0.0.8 has released, so you can install it from vscode extensions marketplace

**Comment by ABA-cedricG** [comment]
ctrl + shift +p, still no "nyan" commands

**Comment by lillwimberly** [comment]
same issue as before, this is tragic.

**Comment by mathric** [comment]
After playing with the source code a little bit, there probably some places but not all will need to update if want to run on newer vscode. I can't make it work at the end, but here's what I found.
- Looks like the extension fail to load on the first place due to the workbench.html path has been updated from
vs\code\electron-browser\workbench\workbench.html
to
vs\code\electron-sandbox\workbench\workbench.html

- svg in readme might cause some problem

- I also got the error that the workbench.html refused to execute inline config inject script because it violates the Content Security Policy directive

- vscode node module is deprecated now, not sure the impact but I replaced it with @types/vscode to run the vsce.
